### DISCUSSION TITLE
Using {{ $value }} in description doesnt work

### DISCUSSION THREAD

**Discussion by techwavedev** [grafana/grafana#37394] [Alerting]
Hi all, 

I'm not sure if this is a bug or not but I'm  not able to use {{ $value }} at ngalert description it always return me:

lvl=eror msg="error in expanding template" logger=ngalert name=description value="Value is expected:  {{ $value }}" err="error parsing template __alert_Under_Replicated_Partitions: template: __alert_Under_Replicated_Partitions:1: **undefined variable \"$value\""**

![image](https://user-images.githubusercontent.com/36747067/127636198-6261df98-845c-442f-9b2a-8d2d4cacc63f.png)

How is the right way of having the values resulted in the expression or query on the alert description? 

Kind Regards,

**Comment by tom0010** [comment]
I'm also getting the same thing when using grafana cloud and using a Cortex / Loki alert.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
